examples/jsm/renderers/common/nodes/Nodes.js

```diff
@@ -15,6 +15,73 @@ class Nodes extends DataMap {
 		this.nodeFrame = new NodeFrame();
 		this.nodeBuilderCache = new Map();
 		this.callHashCache = new ChainMap();
+		this.groupsData = new ChainMap();
+
+	}
+
+	updateGroup( nodeUniformsGroup ) {
+
+		const groupNode = nodeUniformsGroup.groupNode;
+		const name = groupNode.name;
+
+		// objectGroup is every updated
+
+		if ( name === objectGroup.name ) return true;
```

Comment:
Why we can't just compare `groupNode === objectGroup` instead of comparing names?

Reply:
It is possible that more than one `sharedUniformGroup` node was created with the same name, this would unify them.